src/GroupTypeManager.php

```diff
@@ -233,6 +233,35 @@ public function getAllGroupBundles($entity_type = NULL) {
   }
 
   /**
+   * Returns a list of all group content bundles keyed by entity type.
+   *
+   * This will return a simple list of group content bundles. If you need
+   * information about the relations between groups and group content bundles
+   * then use getGroupRelationMap() instead.
+   *
+   * @param string $entity_type_id
+   *   Optional entity type ID to filter the bundles by. If omitted bundles of
```

Comment:
maybe split to two different methods - `getAllGroupContentBundles` and `getGroupContentBundlesBy EntityType`?

My reason is, that if you think you have an entity_type_id, but in fact you have a NULL, nothing warns you, and you get a "weird " answer.